kofta/src/app/components/LanguageSelector.tsx

```diff
@@ -70,6 +70,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
 
     /* Other languages */
     { value: "en-PIRATE", label: "🏴‍☠️ Pirate" },
+    { value: "au", label: "🇦🇺 uɐᴉꞁɐɹʇsnⱯ" }, // Australian
```

Comment:
Not sure if it makes more sense to have the flag at the end instead of the beginning